dev/org.eclipse.codewind.openapi.ui/src/org/eclipse/codewind/openapi/ui/Constants.java

```diff
@@ -93,7 +93,7 @@ public Constants() {
             "jaxrs-resteasy",
             "jaxrs-resteasy-eap",
             "spring"},
-        {"Node.js", "nodejs-server"},
+        {"Node.js", "nodejs-express-server", "nodejs-server-deprecated"},
```

Comment:
Why do we need to add the nodejs-server-deprecated? If it is going (or is) deprecated, shouldn't we stop it from showing?

Reply:
If the generator keeps it, we'll keep it.  We try to mirror what the command line generator has.  Another reason is that if someone wants to use it, they don't have to get a back-level version of the generator.